src/math/lp/lp_bound_propagator.h

```diff
@@ -278,35 +233,74 @@ class lp_bound_propagator {
         }
         return false;
     }
+
+    // pol for polarity
+    int pol(const vertex* v) const { return pol(v->column()); }
+    int pol(unsigned j) const { return m_pol[j].pol(); }
```

Comment:
is polarity a sign as in util/sign.h?

Reply:
Yes, it is almost the same, but cannot be zero. I am not sure using sign.h will simplify things.
For example, the code 
` if (pol(v->column()) == -1)`
would be replaced by 
`if (pol(v->column() == sign_neg)`.